leptos/src/lib.rs

```diff
@@ -20,76 +20,76 @@
+//! [`error_boundary`]: https://github.com/leptos-rs/leptos/tree/main/examples/error_boundary
+//! [`parent_child`]: https://github.com/leptos-rs/leptos/tree/main/examples/parent_child
+//! [`fetch`]: https://github.com/leptos-rs/leptos/tree/main/examples/fetch
+//! [`router`]: https://github.com/leptos-rs/leptos/tree/main/examples/router
+//! [`slots`]: https://github.com/leptos-rs/leptos/tree/main/examples/slots
+//! [`spread`]: https://github.com/leptos-rs/leptos/tree/main/examples/spread
+//! [`counter_isomorphic`]: https://github.com/leptos-rs/leptos/tree/main/examples/counter_isomorphic
+//! [`todomvc`]: https://github.com/leptos-rs/leptos/tree/main/examples/todomvc
+//! [`hackernews`]: https://github.com/leptos-rs/leptos/tree/main/examples/hackernews
+//! [`hackernews_axum`]: https://github.com/leptos-rs/leptos/tree/main/examples/hackernews_axum
+//! [`todo_app_sqlite`]: https://github.com/leptos-rs/leptos/tree/main/examples/todo_app_sqlite
+//! [`todo_app_sqlite_axum`]: https://github.com/leptos-rs/leptos/tree/main/examples/todo_app_sqlite_axum
+//! [`tailwind`]: https://github.com/leptos-rs/leptos/tree/main/examples/tailwind_csr
 //!
 //! Details on how to run each example can be found in its README.
 //!
 //! # Quick Links
 //!
 //! Here are links to the most important sections of the docs:
-//! - **Reactivity**: the [`leptos_reactive`] overview, and more details in
-//!   - signals: [`create_signal`], [`ReadSignal`], and [`WriteSignal`] (and [`create_rw_signal`] and [`RwSignal`])
-//!   - computations: [`create_memo`] and [`Memo`]
-//!   - `async` interop: [`create_resource`] and [`Resource`] for loading data using `async` functions,
-//!     and [`create_action`] and [`Action`] to mutate data or imperatively call `async` functions.
-//!   - reactions: [`create_effect`]
-//! - **Templating/Views**: the [`view`] macro
+//! - **Reactivity**: the [`reactive_graph`] overview, and more details in
+//!   - signals: [`signal`](leptos::prelude::signal), [`ReadSignal`](leptos::prelude::ReadSignal),
+//!   [`WriteSignal`](leptos::prelude::WriteSignal) and [`RwSignal`](leptos::prelude::RwSignal).
+//!   - computations: [`Memo`](leptos::prelude::Memo).
+//!   - `async` interop: [`Resource`](leptos::prelude::Resource) for loading data using `async` functions
+//!     and [`Action`](leptos::prelude::Action) to mutate data or imperatively call `async` functions.
+//!   - reactions: [`Effect`](leptos::prelude::Effect) and [`RenderEffect`](leptos::prelude::RenderEffect).
+//! - **Templating/Views**: the [`view`] macro and [`IntoView`](leptos::IntoView) trait.
 //! - **Routing**: the [`leptos_router`](https://docs.rs/leptos_router/latest/leptos_router/) crate
-//! - **Server Functions**: the [`server`](crate::leptos_server) macro, [`create_action`], and [`create_server_action`]
+//! - **Server Functions**: the [`server`](macro@leptos::prelude::server) macro and [`ServerAction`](leptos::prelude::ServerAction).
```

Comment:
`server` is ambiguous too but I think that it only applies in the context of rustdoc.